ipam/testutils_test.go

```diff
@@ -128,7 +129,7 @@ type mockDB struct{}
 func (d *mockDB) Load(_ string, _ interface{}) (bool, error) { return false, nil }
 func (d *mockDB) Save(_ string, _ interface{}) error         { return nil }
 
-func makeAllocator(name string, cidrStr string, quorum uint) (*Allocator, address.CIDR) {
+func makeAllocator(name string, cidrStr string, quorum uint, mon monitor.Monitor) (*Allocator, address.CIDR) {
```

Comment:
If you introduce the `WithMonitor` variant here, then you don't have to go round adding the null one in so many places.